common/dynamic_country_names/imperia_dynamic_country_names.txt

```diff
@@ -1,217 +1,210 @@
-
-		is_main_tag_only = yes # default no, if yes then only the primary country for a definition can use this name (ie Poland, not dynamic Silesia-Poland)
-		priority = 1 # default 0, if multiple names have valid triggers, higher priority is used. If same priority, first one in file is used
-		
-		trigger = { # Scripted trigger, country-scope
+		name = mch_eastern_provinces		# This can be left empty, and then default tag name will be used
+		adjective = mch_eastern_provinces_adj		# This can be left empty, and then default tag adjective will be used
+		is_main_tag_only = yes		# default no, if yes then only the primary country for a definition can use this name (ie Poland, not dynamic Silesia-Poland)
+		priority = 1		# default 0, if multiple names have valid triggers, higher priority is used. If same priority, first one in file is used
+		trigger = {
+			# Scripted trigger, country-scope
 			exists = c:CHI
 			OR = {
-				c:CHI = {
-				has_diplomatic_pact = {
-					who = c:MCH
-					type = puppet
-					is_initiator = yes
+				c:CHI ?= {
+					has_diplomatic_pact = {
+						who = c:MCH
+						type = puppet
+						is_initiator = yes
 					}
 				}
-				c:CHI = {
-				has_diplomatic_pact = {
-					who = c:MCH
-					type = vassal
-					is_initiator = yes
+				c:CHI ?= {
+					has_diplomatic_pact = {
+						who = c:MCH
+						type = vassal
+						is_initiator = yes
 					}
 				}
-				c:CHI = {
-				has_diplomatic_pact = {
-					who = c:MCH
-					type = dominion
-					is_initiator = yes
+				c:CHI ?= {
+					has_diplomatic_pact = {
+						who = c:MCH
+						type = dominion
+						is_initiator = yes
 					}
 				}
-			}			
+			}
 		}
 	}
 }
 
 MGL = {
 	dynamic_country_name = {
-		name = mgl_banners # This can be left empty, and then default tag name will be used
-		adjective = mgl_banners_adj # This can be left empty, and then default tag adjective will be used
-
-		is_main_tag_only = yes # default no, if yes then only the primary country for a definition can use this name (ie Poland, not dynamic Silesia-Poland)
-		priority = 1 # default 0, if multiple names have valid triggers, higher priority is used. If same priority, first one in file is used
-		
-		trigger = { # Scripted trigger, country-scope
-		exists = c:CHI
-			c:CHI = {
+		name = mgl_banners		# This can be left empty, and then default tag name will be used
+		adjective = mgl_banners_adj		# This can be left empty, and then default tag adjective will be used
+		is_main_tag_only = yes		# default no, if yes then only the primary country for a definition can use this name (ie Poland, not dynamic Silesia-Poland)
+		priority = 1		# default 0, if multiple names have valid triggers, higher priority is used. If same priority, first one in file is used
+		trigger = {
+			# Scripted trigger, country-scope
+			exists = c:CHI
+			c:CHI ?= {
 				has_diplomatic_pact = {
-				who = c:MCH
-				type = vassal
-				is_initiator = yes
+					who = c:MGL
```

Comment:
c:MGLshould be scope:actor and you should check whether it exists at the beginning (like c:CHI)